.github/workflows/rust.yml

```diff
@@ -241,9 +229,14 @@ jobs:
       - name: Cache Cargo
         uses: actions/cache@v3
         with:
-          path: /github/home/.cargo
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            ./target/
           # this key equals the ones on `linux-build-lib` for re-use
-          key: cargo-cache-
+          key: cargo-cache-benchmark-${{ hashFiles('datafusion/**/Cargo.toml', 'benchmarks/Cargo.toml') }}
```

Comment:
The key above also includes `'datafusion-cli/Cargo.toml'` -- is in intentional that these keys do not match?

```
          key: cargo-cache-benchmark-${{ hashFiles('datafusion/**/Cargo.toml', 'benchmarks/Cargo.toml', 'datafusion-cli/Cargo.toml') }}
```

Reply:
Yes, it's intended.
This job (verify benchmark results) seems not to touch `datafusion-cli`, so this cache should not be stale on changing `datafusion-cli/Cargo.toml`.